snap/snapcraft.yaml

```diff
@@ -38,6 +38,8 @@ plugs:
     interface: system-files
     write:
       - /sys/kernel/debug/sched/features
+  shared-memory:
+    private: true
```

Comment:
The [top level plug](https://snapcraft.io/docs/snapcraft-yaml-schema#plugs) is needed because it has configurations. But this access isn't needed by all the apps within this snap. Please assign it to the `cyclictest` app.

Reply:
Fixed on https://github.com/canonical/rt-tests-snap/pull/44/commits/269a0808c4c891b29ad9be0ea41dff851dc44dcc. Thanks.